chainerx_cc/chainerx/routines/evaluation.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <cstdint>
+
+#include <absl/types/optional.h>
+
+#include "chainerx/array.h"
+
+namespace chainerx {
+
+Array Accuracy(const Array& x1, const Array& x2, const absl::optional<int8_t>& ignore_label);
```

Comment:
Hm, is `int8_t` enough (max 127)? How about using `int64_t` since converting this primitive to a `Scalar` will promote it to an `int64_t` anyway and there shouldn't be any significant performance penalty?

Reply:
Thanks. Made the fix.